### ISSUE TITLE
File replciation

### ISSUE THREAD

**Issue by gedw99** [maxpert/marmot#24]
Pocketbase supports files. Store the existence in SQLite and the actual on the File system.

We can use NATS object store with chunking to replicate files to then.

https://github.com/mprimi/nasefa for inspiration of the concept.

**Comment by maxpert** [comment]
I believe this can be separate script that can listen to Marmot changes getting picked up and then uploading them correctly whatever target storage one might deserve. Let's stick to UNIX philosophy and not a generic bloat tool with specific scenarios. Marmot should not and will not understand if something is file path, rather it will provide you with CDC pipeline.

**Comment by gedw99** [comment]
Yep its just a matter of working out how to expose this as a "BackPlane" of sorts for others to tap into it.

I will look into using https://github.com/seaweedfs/seaweedfs
It uses SQLite in the same way that pocketbase uses it. 
Marmot is a very good match for it.
